tests/storage/cdi_import/test_import_http.py

```
@@ -444,10 +440,10 @@ def test_certconfigmap_missing_or_wrong_cm(data_volume_multi_storage_scope_funct
 )
 @pytest.mark.s390x
 def test_successful_concurrent_blank_disk_import(
-    dv_list_created_by_multiprocess,
-    vm_list_created_by_multiprocess,
+    blank_disk_dv_list,
```

Ruff (0.14.14)

[warning] Unused function argument: `blank_disk_dv_list`

(ARG001)